core/src/main/java/io/undertow/server/session/InMemorySessionManager.java

```diff
@@ -56,7 +57,7 @@
 
     private final SessionIdGenerator sessionIdGenerator;
 
-    private final ConcurrentMap<String, SessionImpl> sessions;
```

Comment:
This change might not be required. ConcurrentMap still might be better since it does not throw concurrent modification exception.